### PR TITLE
Add start and end timestamp args to GetTrace request

### PR DESCRIPTION
**Pull request by alburthoffman**
## Which problem is this PR solving?
- https://github.com/jaegertracing/jaeger/issues/4150

## Short description of the changes
- allow optional time parameters when querying trace by id

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Please make sure commits are signed.

**Comment by alburthoffman** [comment]
sure. something wrong with my local env. will fix and sign the commit.

**Review by yurishkuro — APPROVED** [review]

File: proto/api_v3/query_service.proto [hunk 1]
```diff
@@ -27,6 +27,10 @@ option java_package = "io.jaegertracing.api_v3";
 message GetTraceRequest {
   // Hex encoded 64 or 128 bit trace ID.
   string trace_id = 1;
+  // Optional. The start time to search trace id
+  optional google.protobuf.Timestamp start = 2;
+  // Optional. The end time to search trace id
+  optional google.protobuf.Timestamp end = 3;
```
Comment (yurishkuro) [inline proto/api_v3/query_service.proto:33]: CI fails on the `optional` keyword. IIRC all fields are optional by default, so let's drop the keyword.

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Do you only need this change in V3? Your original ticket mentioned Grafana as UI, does it query Jaeger using V3 API (OTLP-based)?

**Pull request merged**